tests/dragonfly/utility.py

```diff
@@ -274,7 +274,7 @@ def size_change_probs(self):
         # Shrink has a roughly twice as large expected number of changed keys than grow
         return [
             max(self.base_diff_prob - self.diff_speed * dist, self.min_diff_prob),
-            1.0,
+            15.0,
```

Comment:
increasing the probability for change commands ,reduces the probability for shrink (del) commands when reaching the state where we reach the target keys.
The reason for this change:
When running replications tests and comparing master to replica at the end of the test, if lots of keys were deleted while running the test we might miss bugs in replicaiton because many keys that were added/updated while seeder was running are not checked at the end

Reply:
That's a significant change, it will make growth much slower. But let's make it 🤷🏻‍♂️ 

In that part I like SeederV2s model more as it's not linear https://github.com/dragonflydb/dragonfly/pull/2556. BTW, Please use it instead of the old one for new tests 🙂